main/src/ca/uwaterloo/flix/language/phase/jvm/GenEffectClasses.scala

```diff
@@ -86,7 +86,7 @@ object GenEffectClasses {
     mv.visitEnd()
   }
 
-  private def genFieldAndMethod(visitor: ClassWriter, effectName: JvmName, op: Op): Unit = {
```

Comment:
These implicit root arguments are used to convert Enum types. Now that we don't assume erased types, these are required (but unused currently because the types are indeed erased)